client/lib/keyboard-shortcuts/index.js

```diff
@@ -12,18 +14,12 @@ const keymaster = typeof window === 'undefined' ? undefined : require( 'keymaste
  * Internal dependencies
  */
 import Emitter from 'lib/mixins/emitter';
-import KeyBindings from 'lib/keyboard-shortcuts/key-bindings';
-
-/**
- * Module variables
- */
-let flatKeyBindings = [];
-const keyBindings = KeyBindings.get();
+import KEY_BINDINGS from './key-bindings';
 
-// flatten the key-bindings object to create an array of key-bindings
-Object.keys( keyBindings ).forEach( function( category ) {
-	flatKeyBindings = flatKeyBindings.concat( keyBindings[ category ] );
-} );
+// Flatten the key-bindings object to create an array of key-bindings. `_.flatMap` converts
+// object in shape `{ c1: [ k1, k2 ], c2: [ k3, k4 ] }` to `[ k1, k2, k3, k4 ]`, i.e., flat-maps
+// over the object property values.
+const flatKeyBindings = flatMap( KEY_BINDINGS );
```

Comment:
:heart:

Reply:
Oh yeah, Lodash can be really cool sometimes 🥇